feature/experimental/p4rt/ate_tests/lldp_packetout_test/lldp_packetout_test.go

```diff
@@ -0,0 +1,450 @@
+
+	p2 := dut.Port(t, "port2")
+	i2 := &oc.Interface{Name: ygot.String(p2.Name())}
+	gnmi.Replace(t, dut, d.Interface(p2.Name()).Config(), configInterfaceDUT(i2, &dutPort2))
+}
+
+// configureATE configures port1 and port2 on the ATE.
+func configureATE(t *testing.T, ate *ondatra.ATEDevice) *ondatra.ATETopology {
+	top := ate.Topology().New()
+
+	p1 := ate.Port(t, "port1")
+	i1 := top.AddInterface(atePort1.Name).WithPort(p1)
+	i1.IPv4().
+		WithAddress(atePort1.IPv4CIDR()).
+		WithDefaultGateway(dutPort1.IPv4)
+
+	p2 := ate.Port(t, "port2")
+	i2 := top.AddInterface(atePort2.Name).WithPort(p2)
+	i2.IPv4().
+		WithAddress(atePort2.IPv4CIDR()).
+		WithDefaultGateway(dutPort2.IPv4)
+
+	return top
+}
+
+// configureDeviceID configures p4rt device-id on the DUT.
+func configureDeviceID(ctx context.Context, t *testing.T, dut *ondatra.DUTDevice) {
+	component := oc.Component{}
+	component.IntegratedCircuit = &oc.Component_IntegratedCircuit{}
+	component.Name = ygot.String(*p4rtNodeName)
+	component.IntegratedCircuit.NodeId = ygot.Uint64(deviceID)
+	gnmi.Replace(t, dut, gnmi.OC().Component(*p4rtNodeName).Config(), &component)
+}
+
+// configurePortID configures p4rt port-id on the DUT.
+func configurePortID(ctx context.Context, t *testing.T, dut *ondatra.DUTDevice) {
+	ports := sortPorts(dut.Ports())
+	for i, port := range ports {
+		gnmi.Replace(t, dut, gnmi.OC().Interface(port.Name()).Id().Config(), uint32(i)+portID)
+	}
+}
+
+// setupP4RTClient sends client arbitration message for both leader and follower clients,
+// then sends setforwordingpipelineconfig with leader client.
+func setupP4RTClient(ctx context.Context, args *testArgs) error {
+	// Setup p4rt-client stream parameters
+	streamParameter := p4rt_client.P4RTStreamParameters{
+		Name:        streamName,
+		DeviceId:    deviceID,
+		ElectionIdH: uint64(0),
+		ElectionIdL: electionID,
+	}
+
+	// Send ClientArbitration message on both p4rt leader and follower clients.
+	clients := []*p4rt_client.P4RTClient{args.leader, args.follower}
+	for index, client := range clients {
+		if client != nil {
+			client.StreamChannelCreate(&streamParameter)
+			if err := client.StreamChannelSendMsg(&streamName, &p4v1.StreamMessageRequest{
+				Update: &p4v1.StreamMessageRequest_Arbitration{
+					Arbitration: &p4v1.MasterArbitrationUpdate{
+						DeviceId: streamParameter.DeviceId,
+						ElectionId: &p4v1.Uint128{
+							High: streamParameter.ElectionIdH,
+							Low:  streamParameter.ElectionIdL - uint64(index),
+						},
+					},
+				},
+			}); err != nil {
+				return errors.New("Errors seen when sending ClientArbitration message.")
+			}
+			if _, _, arbErr := client.StreamChannelGetArbitrationResp(&streamName, 1); arbErr != nil {
+				return errors.New("Errors seen in ClientArbitration response.")
+			}
+		}
+	}
+
+	// Load p4info file.
+	p4Info, err := utils.P4InfoLoad(p4InfoFile)
+	if err != nil {
+		return errors.New("Errors seen when loading p4info file.")
```

Comment:
https://github.com/golang/go/wiki/CodeReviewComments#error-strings

Error strings should not be capitalized (unless beginning with proper nouns or acronyms) or end with punctuation